sandbox/index.ts

```diff
@@ -7,6 +7,9 @@ import Kongponents from '../src'
 import { SandboxLayout } from '@kong-ui-public/sandbox-layout'
 import type { SandboxNavigationItem } from '@kong-ui-public/sandbox-layout'
 import '@kong-ui-public/sandbox-layout/dist/style.css'
+import { v4 as uuidv4 } from 'uuid'
+
+const kongponentsSandboxId = () => `kongponents-sandbox-${uuidv4()}`
```

Comment:
suggestion: we typically won't want a prefix before the UUID since we _already_ usually prefix them in the Kongponent
```suggestion
const kongponentsSandboxId = uuidv4
```

Reply:
ah ok